### PR TITLE
Codeowner transfer from rt4914 to BenHenning till 25th April

### PR DESCRIPTION
**Pull request by rt4914**
I will be busy till 25th April. So transferring my code-ownership to @BenHenning

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -54,9 +54,9 @@ gradlew.bat @BenHenning
 *Test.kt @anandwana001
 
 # All resource files.
```
Comment (BenHenning) [inline .github/CODEOWNERS:56]: For this & other lines, can you add TODOs like:

``TODO(#issue): Replace BenHenning with rt4914 after 25-04-2021.``

or

``TODO(#issue): Re-add rt4914 after 25-04-2021.`` (for items where we share ownership normally).

This also requires filing an issue to tracking re-adding you.

This is all convention from Oppia web to help ensure we don't forget re-adding you. :)

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @rt4914. Just had one comment--PTAL.

File: .github/CODEOWNERS (hunk 1 above)
Comment (rt4914) [reply .github/CODEOWNERS:56]: Done. Thanks.

**Review by BenHenning — APPROVED** [review]
Thanks @rt4914! This LGTM.

**Comment by BenHenning** [comment]
(Re-running another Gradle flake & enabling auto-merge).

**Pull request merged**